zebra-network/src/address_book.rs

```diff
@@ -304,6 +304,8 @@ impl AddressBook {
         self.by_addr
```

Comment:
Usability: Do we need to warn users if they have no recent peers?
Are our existing warnings enough here?

Reply:
I think this might be something for a separate issue?

Reply:
Since the existing logs and metrics use `is_ready_for_connection_attempt`, most of them will be handled by the changes in commit 01d63d109500baa7056c8200983c78de4749dd3e.

I also added this to ticket #1873, because we'll have a better idea what we need to log after we've implemented peer deletion.